input/postgres/vacuum_stats.go

```diff
@@ -84,6 +84,10 @@ func GetVacuumStats(logger *util.Logger, db *sql.DB, ignoreRegexp string) (repor
 		}
 	}
 
+	if err = configRows.Err(); err != nil {
```

Comment:
A separate pre-existing problem, but we should be checking the error returned from `Scan` above, no?

Reply:
thanks for pointing this out, I noticed this but got lazy and wasn't fixing it... definitely better to fix now 👍 updated